app/assets/javascripts/protocols/import_export/import.js.erb

```diff
@@ -495,6 +514,11 @@ function importProtocolFromFile(
       },
       error: function(ev) {
         resultCallback(ev.responseJSON);
+
+        var path = new RegExp("modules");
```

Comment:
Since it's `erb` file, I guess you could use rails here, but to each its own. :)